code/game/objects/items/weaponry.dm

```diff
@@ -617,6 +617,92 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	desc = "A bust of the famous Greek physician Hippocrates of Kos, often referred to as the father of western medicine."
 	icon_state = "hippocratic"
 	impressiveness = 50
+	// If it hits the prob(reference_chance) chance, this is set to TRUE. Adds medical HUD when wielded, but has a 10% slower attack speed and is too bloody to make an oath with.
+	var/reference = FALSE
+	// Chance for above.
+	var/reference_chance = 1
+
+// Coroners are more likely to be evil!
+/obj/item/statuebust/hippocratic/coroner
+	reference_chance = 5
+
+/obj/item/statuebust/hippocratic/Initialize(mapload)
+	. = ..()
+	if(prob(reference_chance))
+		name = "Solemn Vow"
+		desc = "Art lovers will cherish the bust of Hippocrates, commemorating a time when medics still thought doing no harm was a good idea."
+		attack_speed = CLICK_CD_SLOW
+		reference = TRUE
+
+/obj/item/statuebust/hippocratic/examine(mob/user)
+	. = ..()
+	if(reference)
+		. += span_notice("You could activate the bust in-hand to swear or forswear a Hippocratic Oath... but it seems like somebody decided it was more of a Hippocratic Suggestion. This thing is caked with bits of blood and gore.")
+		return
+	. += span_notice("You can activate the bust in-hand to swear or forswear a Hippocratic Oath! This has no effects except pacifism or bragging rights. Does not remove other sources of pacifism. Do not eat.")
+
+/obj/item/statuebust/hippocratic/equipped(mob/living/carbon/human/user, slot)
+	..()
+	if(!(slot & ITEM_SLOT_HANDS))
+		return
+	var/datum/atom_hud/our_hud = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED]
+	our_hud.show_to(user)
+	ADD_TRAIT(user, TRAIT_MEDICAL_HUD, type)
+
+/obj/item/statuebust/hippocratic/dropped(mob/living/carbon/human/user)
+	..()
+	if(HAS_TRAIT_NOT_FROM(user, TRAIT_MEDICAL_HUD, type))
+		return
+	var/datum/atom_hud/our_hud = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED]
+	our_hud.hide_from(user)
+	REMOVE_TRAIT(user, TRAIT_MEDICAL_HUD, type)
+
+/obj/item/statuebust/hippocratic/attack_self(mob/user)
```

Comment:
Is this just meant to be a reference or is there any point to this?

Reply:
reference